src/app/shared/components/course-form/course-form.component.ts

```diff
@@ -31,7 +31,7 @@ export class CourseFormComponent implements OnInit, OnDestroy {
   courseForm!: FormGroup;
   private subscription: Subscription | undefined;
   private allAuthors: Author[] = [];
```

Comment:
This property is unused as of now. Delete

Reply:
Thank you ! Removed